yt_dlp/extractor/crunchyroll.py

```diff
@@ -119,7 +121,7 @@ class CrunchyrollBetaIE(CrunchyrollBaseIE):
             'season_number': 1,
             'episode': 'To the Future',
             'episode_number': 73,
-            'thumbnail': r're:^https://www.crunchyroll.com/imgsrv/.*\.jpeg$',
+            'thumbnail': r're:^https://www.crunchyroll.com/imgsrv/.*\.jpeg?$',
```

Comment:
What's the point of this `?`? It seems like that would make it also match `.jpe`, which I can't see the purpose of.

Same for line 145 below.

Reply:
For me, the responses were giving me `.jpe` thumbnails for some strange reason (on all endpoints as well). Checking in the browser both work (but only those two variations). Figured I'd add them to make tests passing:
https://www.crunchyroll.com/imgsrv/display/thumbnail/60x90/catalog/crunchyroll/708ce7abe9c62bfd35139acaedc32ed1.jpeg
https://www.crunchyroll.com/imgsrv/display/thumbnail/60x90/catalog/crunchyroll/708ce7abe9c62bfd35139acaedc32ed1.jpe

Reply:
... weird. Ok, if they say so, I guess.

Reply:
`.jpg` also seems to work. I just got `.jpe` responses. Interesting...